tests/Rector/Class_/MessageSubscriberInterfaceToAttributeRector/Fixture/messagesubscriber-yield.php.inc

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Rector\Symfony\Tests\Rector\Class_\MessageSubscriberInterfaceToAttributeRector\Fixture;
+
+use App\Message\OtherSmsNotification;
+use App\Message\SmsNotification;
+use Symfony\Component\Messenger\Handler\MessageSubscriberInterface;
+
+class SmsNotificationHandler implements MessageSubscriberInterface
+{
+    public function __invoke(SmsNotification $message)
+    {
+        // ...
+    }
+
+    public function handleOtherSmsNotification(OtherSmsNotification $message)
+    {
+        // ...
+    }
+
+    public static function getHandledMessages(): iterable
+    {
+        // handle this message on __invoke
+        yield SmsNotification::class;
+
+        // also handle this message on handleOtherSmsNotification
+        yield OtherSmsNotification::class => [
+            'method' => 'handleOtherSmsNotification',
+            'priority' => 0,
+            'bus' => 'messenger.bus.default',
```

Comment:
What if we have different bus set? I dont see you do anything with that in `MessageSubscriberInterfaceToAttributeRector` you can also add test fixture for this.

Reply:
Shouldn't be a problem, the arrays are being converted as is. Adding an extra fixture for a different bus seems unnecessary.